### DISCUSSION TITLE
The icon has disappeared.

### DISCUSSION THREAD

**Discussion by luckfunc** [community/community#64763] [Mobile]
The icon for personal profile is missing after entering the homepage![image](https://github.com/community/community/assets/85545304/845baa83-bc4c-4e29-ad11-722747f76ed8)
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.127.0 (107661963)
Device: iPhone 11 (iOS 15.3)
</details>

**Comment by arafajs** [comment]
1. Clear the cache and data for the GitHub app
2. Log out of the GitHub app and log back in.
3. Update the GitHub app to the latest version.

**Reply by luckfunc** [reply]
I tried theme all once,and again

**Reply by luckfunc** [reply]
Thanks

**Reply by arafajs** [reply]
does it work? @Ddkkkk11

**Reply by luckfunc** [reply]
> does it work? @Ddkkkk11 

No, Thank you for your help

**Comment by alriyadsouza** [comment]
Test on Different Devices: I guess the device your using might be facing this issue try on some other tablet or phone. 

Check GitHub Status: It's possible that GitHub might be experiencing temporary technical issues. Check the GitHub Status page (https://www.githubstatus.com/) to see if there are any ongoing incidents that could be affecting the display of icons.

Logout and Login: If none of the above steps work, try logging out of your GitHub account and then logging back in. This might refresh your session and resolve any minor glitches.

**Reply by luckfunc** [reply]
Thanks
1. This is my github status image
![image](https://github.com/community/community/assets/85545304/c3806af3-ffdc-4575-bd6b-38f3d46e3ea8)
2. I tried deferent device,it'good.
![image](https://github.com/community/community/assets/85545304/ddbf1cee-6176-4d8a-9be3-cf9bace89b6f)

I think the github app doesn't have ios15-compatible devices

**Reply by alriyadsouza** [reply]
welcome :)

**Comment by ejrv** [comment]
The server is down currently

**Comment by capiet** [comment]
The problem still exists. iOS 15
![image](https://github.com/community/community/assets/79054211/c1788a10-5f48-41a8-a03e-2fc426c015f4)

**Comment by capiet** [comment]
Are there any update on this bug? It looks like it's not going to be resolved ever.

**Comment by mishmanners** [comment]
This appears to be an iOS issue. We've got a new update for the app. If it's still occurring, which it looks like it is, I've added the "bug" label, and hopefully the Mobile team will be able to reach out and see if there's a fix.

**Comment by alcere** [comment]
Hi @luckfunc @capiet! We apologize for the inconvenience caused. We have opened an internal issue to investigate this issue further. Please stay tuned for updates, and thank you for bringing this to our attention.

**Reply by capiet** [reply]
@mishmanners @alcere Thank you for the response! I am looking forward to get updates about this bug.
